test/converters/converters.go

```diff
@@ -0,0 +1,21 @@
+package converters
+
+import (
+	"github.com/bitrise-io/steps-deploy-to-bitrise-io/test/converters/junitxml"
+	"github.com/bitrise-io/steps-deploy-to-bitrise-io/test/converters/xcresult"
+)
+
+type Intf interface {
+	SetFiles([]string)
+	XML() ([]byte, error)
+	Detect() bool
+}
+
+var handlers = []Intf{
+	&junitxml.Handler{},
+	&xcresult.Handler{},
+}
+
+func List() []Intf {
+	return handlers
```

Comment:
why do not you make the `handlers` an exported var?

Reply:
I want to keep this var unsettable from outside